prometheus/desc.go

```diff
@@ -26,7 +26,7 @@ import (
 )
 
 var (
-	metricNameRE = regexp.MustCompile(`^[a-zA-Z_][a-zA-Z0-9_:]*$`)
+	metricNameRE = regexp.MustCompile(`^[a-zA-Z_:][a-zA-Z0-9_:]*$`)
```

Comment:
We should just use https://github.com/prometheus/common/blob/master/model/metric.go#L25 , now that it is fixed (i.e. don't set a variable here but just use `model.MetricNameRE` below).

Same for next line, use `model.LabelNameRE`, now that we are on it.